iDiff/cmd/root.go

```diff
@@ -57,22 +59,39 @@ var RootCmd = &cobra.Command{
 		if len(diffArgs) == 0 {
 			diffArgs = allDiffers
 		}
+
+		runtime.GOMAXPROCS(2)
```

Comment:
I think you should probably just leave this set to whatever it defaults to. From the docs:

```
As of Go 1.5, the default value of GOMAXPROCS is the number of CPUs (whatever your operating system considers to be a CPU) visible to the program at startup.
```

So if anything, this will be limiting the execution environment later on.